libparsec/crates/platform_storage/src/native/workspace.rs

```diff
@@ -69,13 +69,16 @@ impl PlatformWorkspaceStorage {
                     let _ = std::fs::create_dir_all(parent);
                 }
 
-                SqliteConnectOptions::new()
+                let mut conn = SqliteConnectOptions::new()
                     .filename(&db_path)
                     .create_if_missing(true)
                     .journal_mode(SqliteJournalMode::Wal)
                     .synchronous(SqliteSynchronous::Normal)
+                    .auto_vacuum(SqliteAutoVacuum::Full)
                     .connect()
-                    .await?
+                    .await?;
+                conn.execute("VACUUM;").await?;
```

Comment:
I'm not sure about running the `VACUUM` at every connection, the docs says it should only when the setting is changed right?

Reply:
Yes, we should not run vacuum each time, it is costly even if the db is "clean"